test/fuzz/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
```

Comment:
Please use PCL's license (BSD-3) using a spdx (eg in filters/include/pcl/filters/experimental/).

Since I was the original author of this, please consider this my permission to relicense it under BSD-3

(same with the other file)